controller/backing_image_manager_controller.go

```diff
@@ -339,9 +339,9 @@ func (c *BackingImageManagerController) cleanupBackingImageManager(bim *longhorn
 			log.WithError(err).Warnf("Failed to launch a gRPC client during cleanup, will skip deleting all files")
 		} else {
 			log.Infof("Start to delete all backing image files during cleanup")
-			for biName := range bim.Status.BackingImageFileMap {
-				if err := cli.Delete(biName); err != nil {
-					log.WithError(err).Warnf("Failed to launch a gRPC client during cleanup, will skip deleting the file for backing image %v", biName)
+			for biName, biFileInfo := range bim.Status.BackingImageFileMap {
+				if err := cli.Delete(biName, biFileInfo.UUID); err != nil {
```

Comment:
If the UUID of the BI is unique, why not just `cli.Delete(biFileInfo.UUID)` only? keeping it for debuggin purpose?

Reply:
Previously, the backing image sync server uses the backing image file path as the unique key. 
To figure out the path, both biName and the biUUID are required.

Later on due to the feature requirement, the sync server has to store the UUID as well. But I didn't switch to using the UUID as the unique key. (the time is limited and the filepath is straightforward hence I am hesitating to modify this.)
If the sync server switches to using UUID only, then there is no need to input biName.

Reply:
Do you think the sync server need to switch to using the UUID?

Reply:
Due to limited time, let's keep it as is first. We can do it as refactoring in the future, that is fine for now.

Reply:
Got it!